cmd/lncli/commands.go

```diff
@@ -766,6 +766,10 @@ var sendPaymentCommand = cli.Command{
 			Name:  "pay_req",
 			Usage: "a zbase32-check encoded payment request to fulfill",
 		},
+		cli.Int64Flag{
+			Name:  "max_fee",
+			Usage: "Maximum fee to pay in millisatoshis.",
```

Comment:
It's a bit odd for this unit to be in `mSAT` when currently we accept/display `SAT` everywhere uniformly within the RPC interface (other than when modifying the current fee schedule). This was a purposeful decision, as atm, I don't see users utilizing the `mSAT` payment granularity at this stage.

Reply:
Thanks for the feedback! I originally had this in satoshis and had commented on this PR "Now that I know that "MilliSatoshi are the native unit of the Lightning Network", should this be changed to MilliSatoshis?" I think my squash removed my comments and questions from the actual code but they should be visible on the PR page.

After changing it internally to use msat, I spoke to you about the fee limit on #dev-help and you said "fee limit should operate on msat, as fees are paid in msat".

Shall I change it to operate internally on msat level, but externally on sat level?